IFC4x3/Properties/p/PlasticLimit_2QRKg6a61Ft9rL$4_1Q7GD/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PlasticLimit_2QRKg6a61Ft9rL4_1Q7GD" Name="PlasticLimit" UniqueId="9a6d4a86-9060-4fdc-9d55-fc4f8168740d" PrimaryDataType="IfcPositiveRatioMeasure" />
```

Comment:
Ratio measure?

Reply:
> Ratio measure?

It is currently defined as IfcPositiveRatioMeasure. Is 0 is a valid value?

Reply:
Description updated according to proposal from domain.

Reply:
> > Ratio measure?
> 
> It is currently defined as IfcPositiveRatioMeasure. Is 0 is a valid value?

0 is not valid.